src/language-css/printer-postcss.js

```diff
@@ -582,7 +582,19 @@ function genericPrint(path, options, print) {
           isNextForKeyword ||
           isNextEachKeyword
         ) {
-          parts.push(line);
+          if (atRuleAncestorNode && atRuleAncestorNode.name === "import") {
```

Comment:
@evilebottnawi I have added some more specs and added more checks in here but I think these checks are too many and the `css_comments` is still different from the original one. Can you tell me how can I make it better ?

Reply:
Alright with comments, don't touch their right now